### PR TITLE
Entebbe

### PR DESCRIPTION
**Pull request by brucellino**
discussion around how to provide feedback during the Ansible hands on course.

### CODE REVIEW

File: handson/Ansible/Ansible Lessons.md [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 This directory contains the lesson plans for the Ansible  part of the bootcamp. Each lesson has it's own directory, which contains the description of the tasks to complete during the lesson, in markdown format, as well as supporting files such as inventory or playboook stubs, _etc_.
 
-In order to do the course, go through the lesson description step by step, completing and ticking off tasks as you go. If there are questions, search for or start a topic on the forum linked to that lesson.
```
Comment (chrohrer) [inline handson/Ansible/Ansible Lessons.md:5]: Why did you remove this? I like the people to go to the forum

**Review by chrohrer — COMMENTED** [review]
Why did you remove the forum part?

File: handson/Ansible/Ansible Lessons.md (hunk 1 above)
Comment (brucellino) [reply handson/Ansible/Ansible Lessons.md:5]: Yes, we want people to go to the forum to discuss the course, but when there are actual errors in the content _in this repo_ then it's best to deal with those using the github tickets, don't you think ?

Comment (chrohrer) [reply handson/Ansible/Ansible Lessons.md:5]: Fair enough

**Comment by brucellino** [comment]
@chrohrer I'm preparing a new bootcamp for the CSIR in two weeks' time. This is just a cleanup/synch PR. No significant changes.
